modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,154 @@
+= Provision a Serverless Cluster
+:description: Learn how to create a Serverless Cloud cluster.
+:page-cloud: true
+:page-beta: true
+
+Redpanda Serverless is the easiest way to test event streaming in the cloud: You can start streaming 30 seconds after clicking to create the cluster. 
+
+[NOTE]
+====
+To use our Beta release, Redpanda issues you enough credits to run Redpanda with reasonable throughput. These credits expire 14 days after you activate your tenant through email verification. For details, see <<Beta credits and limits>>. To ask questions, give feedback, or extend your trial, use the **#serverless** https://redpandacommunity.slack.com/[Community Slack^] channel, or contact https://support.redpanda.com/hc/en-us/requests/new[support^].
```

Comment:
Please refocus to:
- Beta is limited in features (see below)
- Best effort support (see below)
- Start your free trial by filling up the form.
- Free trial details are below